cmd/server/assets/admin/users/show.html

```diff
@@ -0,0 +1,72 @@
+{{define "admin/users/show"}}
+{{$user := .user}}
+{{$stats := .stats}}
+
+<!doctype html>
+<html lang="en">
+<head>
+  {{template "head" .}}
+</head>
+
+<body id="users-show" class="tab-content">
+  {{template "admin/navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>{{$user.Name}}</h1>
+    <p>
+      Here is information about the user.
+    </p>
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Details</div>
+      <div class="card-body">
+        <h6 class="card-title">Name</h6>
```

Comment:
Check the `<hr>` + simplified margins on https://github.com/google/exposure-notifications-verification-server/blob/main/cmd/server/assets/admin/realms/edit.html#L43 if you want to use that here.

Reply:
hows that?